getting_started/setup/setup_ktest_environment.rst

```diff
@@ -115,73 +140,86 @@ mess up what the numbers below point to!
       => 7  menuentry ...
 
    # Attempt to boot menuentry 4 only once
-   grub-reboot 4; grub-editenv list
-      => saved_entry=6
-      => next_entry=4
+   grub-reboot 4
    reboot
 
    # Switch to menuentry number 4 as the new "safe" kernel
-   grub-set-default 4; grub-editenv list
-      => saved_entry=4
+   grub-set-default 4
 
 
 Fedora and derived distributions have a more elaborate system to manage
 "installed" kernels. Instead of extracting ``menuentry`` lines from
 ``/boot/grub/grub.cfg`` with the ``awk`` command above, to list all
-installed kernels use: ``grubby --info=ALL``.
-
-After copying it to ``/boot``/, "install" a new kernel with:
-``grubby --add-kernel /boot/vmlinuz-softest --title=softest``.  Check
-``grubby``'s documentation for more details.
+installed kernels use ``grubby --info=ALL``. 
+Check ``grubby``'s documentation for more details.
+To boot a different kernel just once, use ``grub2-reboot [n]``, where ``[n]`` is the index of the menu entry you'd like to boot.
 
 6. Install openssh-server
 -------------------------
 
 .. code-block:: bash
 
+   # On Ubuntu, you need to install it
    sudo apt-get install openssh-server
+
+   #On Fedora, you just need to enable it
+   sudo systemctl enable sshd
+
+   # On either system, you'll need to update the config
    # Use your editor of choice.
    sudo emacs /etc/ssh/sshd_config
 
-Replace ``PermitRootLogin without-password`` with ``PermitRootLogin yes``
-and save.
+Replace ``#PermitRootLogin prohibit-password`` with ``PermitRootLogin yes``
+(make sure to remove the ``#``) and save.
 
-7. reboot target
+7. Reboot target
 ----------------
 
+Make sure it boots automatically to your safe kernel. It's also recommended to test using grub-reboot to boot the test kernel, then rebooting again to make sure it goes back to the safe kernel.
+
```

Comment:
isnt' grub-reboot a Ubuntu thing? I think on Fedora it's a different comment, isn't it?